torchmultimodal/models/flava/model.py

```diff
@@ -171,7 +170,7 @@ def forward(
                 text_encoding_out[1],
             )
         else:
-            text_outputs = text_encoding_out
+            text_outputs = text_encoding_out  # type: ignore
```

Comment:
why ignore?

Reply:
I was getting mypy type errors from these that I could not work around...